src/subdomains/generic/user/models/bank-data/bank-data.entity.ts

```diff
@@ -57,6 +60,18 @@ export class BankData extends IEntity {
   // --- ENTITY METHODS --- //
 
   activate(): UpdateResult<BankData> {
+    const update: Partial<BankData> = {
+      active: true,
+      label: null,
+      preferredCurrency: null,
+    };
```

Comment:
Why set to `null`?

Reply:
If a user wants to delete his iban and then creates the iban new, it would lead to discussion with some customers, why it was not deleted 😅